examples/client-logger/app-insights-logger/package.json

```diff
@@ -67,10 +67,10 @@
 		"@types/testing-library__jest-dom": "^5.14.5",
 		"copyfiles": "^2.4.1",
 		"cross-env": "^7.0.3",
-		"eslint": "~8.6.0",
-		"eslint-config-prettier": "~8.5.0",
-		"eslint-plugin-jest": "~27.1.5",
-		"eslint-plugin-react": "~7.28.0",
+		"eslint": "~8.50.0",
+		"eslint-config-prettier": "~9.0.0",
```

Comment:
Tangent: I don't see eslint-config-prettier in many packages... should it be?

Reply:
Ahhh, this is probably a leftover from when we moved to using require() on the config so we get node module reolution and can load eslint plugins relative to the shared config instead of relative to the package being checked. Before we did that, all packages had to depend on all eslint plugins. :( But that's no longer true, and I'll remove this and the rushstack one from all packages. I already did it with the jsdoc plugin.

Reply:
Apparently this config is still needed relative to the individual packages. If I remove it, then I need to remove the
"extends prettier" part of each config, and once I do that, a bunch of formatting rules get turned on - I think because
the config is effectively not loaded. @Josmithr Do you know if this is expected?

Reply:
No, that isn't expected 😢